### PR TITLE
Beta/1.1.0/8 -> Develop

### PR DESCRIPTION
**Pull request by jiyoung-an**
### Summary
- upgrade package version to beta.8.
- fix empty metadata issue.
- refactor example scripts for `nft`, `chat`.
- fix duplicate assistant creation.
- set write rule, gc rule. _(will fix related bug in next PR)_
- allow third-party provider. (must ainize supported!)

### To-Do
- [ ] rename `provider` -> `serviceNickname`
- [ ] fix write rule bug
- [ ] check missing gc rule

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/util.ts [hunk 1]
```diff
@@ -195,12 +195,17 @@ export const validateObjectOwner = async (appId: string, address: string, ain: A
...
+  ainize: Ainize
+): Promise<string> => {
   const serviceName = PROVIDER_SERVICE_NAME_MAP.get(provider);
-  if (!serviceName) {
-    throw new Error('Service is currently not supported');
-  }
-  return serviceName;
+  if (serviceName) return serviceName;
+
+  const service = await ainize.getService(provider);
```
Comment (jiyoung-an) [inline src/util.ts:205]: currently if provider name not exists in map, it query on ainize.

File: src/util.ts [hunk 2]
```diff
@@ -329,8 +335,22 @@ export const sendRequestToService = async <T>(
 ): Promise<T> => {
   try {
     await ainizeLogin(ain, ainize);
+
+    // TODO(jiyoung): check status to identify failure (if implemented)
     const data = await service.request({ ...body, jobType });
+    if (data) {
```
Comment (jiyoung-an) [inline src/util.ts:341]: temporary fix.

File: src/util.ts (hunk 1 above)
Comment (platfowner) [reply src/util.ts:205]: If 'provider' could be service name, how about using 'service nickname' or 'service name alias' instead? i.e., PROVIDER_SERVICE_NAME_MAP -> SERVICE_NICKNAME_MAP or SERVICE_NAME_ALIAS_MAP?

Comment (jiyoung-an) [reply src/util.ts:205]: I think 'SERVICE_(NICK)NAME_MAP' is good! 
could using "service name" and "provider" be confusing? what about using "service" only?

Comment (platfowner) [reply src/util.ts:205]: SERVICE_NAME_MAP also looks good to me, thanks!

Comment (jiyoung-an) [reply src/util.ts:205]: thanks to discuss this issue! 
i will apply renaming in next pr, it's hard to see the changes at a glance. (-> todo)

Comment (platfowner) [reply src/util.ts:205]: right, sounds great!

**Comment by jiyoung-an** [comment]
Thanks to reviews and discussion! @platfowner @minsulee2 now merging...

**Pull request merged**